code/obj/item/clothing/gloves.dm

```diff
@@ -239,6 +239,15 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 		qdel(src)
 		user.put_in_hand_or_drop(new /obj/item/clothing/gloves/fingerless)
 	else . = ..()
+
+/obj/item/clothing/gloves/black/ovenmitts
```

Comment:
Dislike `/gloves/black` and them being not that.

Reply:
Not sure what that means. It's a child, so it's just going to inherit the gloves heat protection. It uses a unique name, description and glove fibre